<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormImportBatch</class>
 <widget class="QWidget" name="FormImportBatch">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Importação em Lote</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qdocbase.qrc">
    <normaloff>:/images/png/images/docbase64x64.png</normaloff>:/images/png/images/docbase64x64.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frameToolBar">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="tableWidget_Sheet_P" name="tableWidget_Sheet">
     <column>
      <property name="text">
       <string>Arquivo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Número</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Valor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Id. Documento</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="progressBar_ImportBatch">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tableWidget_Sheet_P</class>
   <extends>QTableWidget</extends>
   <header location="global">tablewidget_sheet_p.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qdocbase.qrc"/>
 </resources>
 <connections/>
</ui>
